catalog/video/video-mpegts-stream/src/main/java/org/codice/alliance/video/stream/mpegts/Context.java

```diff
@@ -30,6 +31,8 @@
 
     private Optional<Metacard> parentMetacard = Optional.empty();
 
+    private final AtomicBoolean isParentDirty = new AtomicBoolean(false);
```

Comment:
Can we rename it to `parentNeedsUpdate` so its purpose is clear to other developers?